### ISSUE TITLE
file  encoding problem in windows

### ISSUE THREAD

**Issue by nuborisar** [appswithlove/loco-android#32]
Hi,
There is a problem when using the plugin on pc windows. Some chars show wrong.

In TaskUtils.groovy
     saveFile(LocoConfig locoConfig...

When you write the file, by not explicitly specifying the encoding to UTF-8 in windows it writes it as ANSI producing wrong characters, for example with accented uppercase.
![image](https://user-images.githubusercontent.com/11457330/167119756-c3cf0f84-74e6-447a-879e-3ef677a63ad9.png)

Changing the file.write to something like:

 ```
Writer out = new BufferedWriter(new OutputStreamWriter(
            new FileOutputStream(directory.absolutePath + "/" + locoConfig.fileName + ".xml"), "UTF-8"));
        try {
            out.write(text);
        } finally {
            out.close();
        }
```
        
maybe fix this problem and other charsetmap problems

I can change it and make a PR, but I have not been able to test it properly and I leave it to your interpretation.

Thank you very much and I hope you can solve this problem.

**Comment by yannickpulver** [comment]
Hi @nuborisar 

Thanks for the detailed description. 
I'll try to check if it works with the code you've provided 👍

**Comment by yannickpulver** [comment]
Hi @nuborisar 
After some delay, we published a new release `0.2.10` that should fix this.
Let me know if you still have issues.

**Comment by nuborisar** [comment]
It works fine in Windows.

Many Thanks!
